### ISSUE TITLE
logged file name

### ISSUE THREAD

**Issue by leggewie** [kdeldycke/mail-deduplicate#157]
#### Preliminary checks

* [x] I am running the latest version
* [x] I have [read the Code of Conduct](https://github.com/kdeldycke/mail-deduplicate/blob/develop/.github/code-of-conduct.md)
* [x] I have checked there is not other [Issues](https://github.com/kdeldycke/mail-deduplicate/issues) or [Pull Requests](https://github.com/kdeldycke/mail-deduplicate/pulls) covering the same topic to open

As you can see in #155 errors in files are being logged with the filename being logged as ```/home/user/Maildir/.Erledigt:1406922416.M923538P22149.giga,W=2249,S=2199```  for example.  That looks like the filepath, but it isn't quite it.  First the path to the directory ends in : and not / as well as missing /cur/.  Furthermore, the filename in this case lacks a few more characters ```:2,S```.  The correct and complete filenpath would be ```/home/user/Maildir/.Erledigt/cur/1406922416.M923538P22149.giga,W=2249,S=2199:2,S```. This thwarts any attempts at copy and pasting the filepath to inspect the file directly.

Am I correct to assume this is not intended behaviour?

**Comment by kdeldycke** [comment]
You're right there is an issue here. As the mail source is a Maildir, we should expect the output to be real, fully-qualified path, and must indeed be allowed to be copy-pasted without effort.

The root cause should be somewhere around these lines: https://github.com/kdeldycke/mail-deduplicate/blob/f423b414c400d21e338edf09bf904d39697fc7b8/mail_deduplicate/mailbox.py#L138-L146 . In which the recursive opening of maildir subfolders happens.

If you could come up with a fix, feel free to propose a PR! :)

**Comment by leggewie** [comment]
I'm not sure about that.  I believe https://github.com/kdeldycke/mail-deduplicate/blob/f423b414c400d21e338edf09bf904d39697fc7b8/mail_deduplicate/mail.py#L93-L94 
is where it's at.

**Comment by leggewie** [comment]
Indeed, https://github.com/leggewie/mail-deduplicate/commit/62e3b1de3e8341c1400ccf13fd4052dce323463a results in a string that can be copied&pasted straight off the terminal.  Obviously, it's not the right fix.

Questions:
* what is responsible for truncating file names after a : colon?
* why does source_path not include {tmp,cur,new}?
* how does this affect mailbox formats other than maildir?

**Comment by kdeldycke** [comment]
We now identify mails from folder-based boxes in logs by the fully-qualified path of their own file. This will be available in the upcoming 9.1.0 release.

**Comment by leggewie** [comment]
Awesome, thank you for your continued work on this fine piece of software.
